dvtemplates/build_k8s_cluster.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -ex
```

Comment:
We're going to want to remove the x on these, to prevent exposing any sensitive data.

Reply:
Done